docs/content/kv_commands.md

```diff
@@ -83,6 +94,7 @@ Reads a single value by key from the given namespace.
 
 ```sh
 $ wrangler kv:key get f7b02e7fc70443149ac906dd81ec1791 "key"
+value
```

Comment:
This looks like I need to enter `value` in my terminal on the next line.

Reply:
I agree. I can make replace this with a more explicit string that signifies "this is a value returned by the API and output to the terminal". Like 
```suggestion
value-returned-from-api-and-output-to-terminal
```

Reply:
or give a classic `=>` return signifier

Reply:
Ooh yes!

Reply:
```suggestion
=> value
```